pytest.ini

```diff
@@ -21,5 +21,6 @@ junit_logging=all
 junit_family=xunit2
 junit_log_passing_tests=0
 asyncio_mode=auto
+log_cli = true
```

Comment:
why do we need `log_cli`?

Reply:
Not necessary but logs with it provide much more info for investigation, at least for UI tests. @mganisin what do you think?

Reply:
good for your local PYTEST_ADDOPTS, not that great as global default. Even if considered good, certainly should not be introduced here, but in separate MR.